lua/neo-tree/defaults.lua

```diff
@@ -257,10 +257,14 @@ local config = {
     last_modified = {
       enabled = true,
       required_width = 88, -- min width of window required to show this column
+      format = "%Y-%m-%d %I:%M %p", -- format string for timestamp (see ISO C function `strftime`, `format` parameter)
+                                    -- or use a function that takes in the date in seconds and returns a string
     },
     created = {
       enabled = false,
       required_width = 120, -- min width of window required to show this column
+      format = "%Y-%m-%d %I:%M %p", -- format string for timestamp (see ISO C function `strftime`, `format` parameter)
+                                    -- or use a function that takes in the date in seconds and returns a string
```

Comment:
There's builtin help for lua's `os.date()` (which also leads to vimscript's `strftime()`) so that might be a bit more convenient to refer to:
```suggestion
      format = "%Y-%m-%d %I:%M %p", -- format string for timestamp (see `:h os.date()`)
                                    -- or use a function that takes in the date in seconds and returns a string
    },
    created = {
      enabled = false,
      required_width = 120, -- min width of window required to show this column
      format = "%Y-%m-%d %I:%M %p", -- format string for timestamp (see `:h os.date()`)
                                    -- or use a function that takes in the date in seconds and returns a string
```

Reply:
Alright, I'll update the documentation.